src/App.css

```diff
@@ -1,25 +1,20 @@
-.navbar {
-  display: flex;
-  justify-content: space-between;
-  background-color: rgb(233, 224, 218);
-  padding: 10px;
-  margin-bottom: 30px;
-}
-
-.navbar ul {
-  list-style: none;
-  display: flex;
+.rangi {
+  color: #0290ff;
+  text-decoration: none;
 }
 
-.navbar ul > li {
-  padding: 5px 10px;
-  font-size: 19px;
+body {
+  margin: 0;
+  padding: 10px;
+  background: #c7c7c7;
 }
 
-.navbar ul > li:last-child {
-  border-right: none;
+.wrapper {
+  width: 85%;
+  margin: auto;
+  background: #fafafa;
 }
 
-.navbar a {
-  color: #25649e;
+ul {
+  background: #fafafa;
 }
```

Comment:
- [ ] As the requirements say you should follow the design on [Zeplin]() that means colors, layouts etc.
- [ ] They are features that were not implemented yet you should add them as placeholders like comments, edit, update progress button etc.
refer to this:
![zp](https://user-images.githubusercontent.com/44745855/135504646-4cfa1bb1-1140-4c9f-a433-7377210a5704.png)

Reply:
Thank you for the review I have implemented the requested changes.